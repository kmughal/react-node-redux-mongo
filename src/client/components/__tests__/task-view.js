import React from "react";
import {
  TasksViewComponent
} from "../../components/tasks-view";
import * as renderer from "react-test-renderer";

describe("The Task View Component.", () => {
      it("render", () => {
          const tree = renderer.create( < TasksViewComponent tasks = {
              []
            }
            />).toJSON();
            expect(tree).toMatchSnapshot();
          })
      })