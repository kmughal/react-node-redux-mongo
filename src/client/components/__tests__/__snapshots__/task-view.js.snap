// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Task View Component. render 1`] = `
<section>
  <button
    aria-controls="collapseExample"
    aria-expanded="false"
    className="btn btn-success"
    data-target="#tasks-list"
    data-toggle="collapse"
    type="button"
  >
    Tasks [
    0
    ]
  </button>
  <div>
    <br />
    <p
      className="alert alert-info"
      role="alert"
    >
      Currently you dont have any task.
    </p>
  </div>
</section>
`;
